RELEASENOTES.md

```diff
@@ -1 +1 @@
-
+- Add additional error handling to mannequin reclaimation process
```

Comment:
```suggestion
- Add additional error handling to `reclaim-mannequin` process
```

Reply:
Updated verbiage in c939758efc03a84ff993b188194864aea7966231